ttl/cache/BUILD.bazel

```diff
@@ -0,0 +1,35 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "cache",
+    srcs = ["table.go"],
+    importpath = "github.com/pingcap/tidb/ttl/cache",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//parser/ast",
+        "//parser/model",
+        "//parser/mysql",
+        "//table/tables",
+        "//ttl/session",
+        "//types",
+        "//util/chunk",
+        "@com_github_pingcap_errors//:errors",
+    ],
+)
+
+go_test(
+    name = "cache_test",
+    srcs = [
+        "main_test.go",
+        "table_test.go",
+    ],
```

Comment:
Please ```flaky = True,``` and the other need to do too.

Reply:
OK.

Is it still needed even when the test is not flaky?

Reply:
Added.

At least, I understand it's better than not.

Reply:
Yes, It is defensive